packages/common/refresh-runtime.js

```diff
@@ -653,10 +653,7 @@ export function validateRefreshBoundaryAndEnqueueUpdate(
 
 function predicateOnExport(ignoredExports, moduleExports, predicate) {
   for (const key in moduleExports) {
-    if (key === '__esModule') continue
     if (ignoredExports.includes(key)) continue
-    const desc = Object.getOwnPropertyDescriptor(moduleExports, key)
-    if (desc && desc.get) return key
```

Comment:
Since `moduleExports` is a module namespace, all properties are not a sideeffectful getter.